### PR TITLE
22.0/potti/react native app update

### PR DESCRIPTION
**Pull request by ChueiZuoChen**

### CODE REVIEW

**Comment by ChueiZuoChen** [comment]
Complete FaceScan and FingerScan

**Comment by ChueiZuoChen** [comment]
PR is ready

File: ReactNative/AHIReactNativeMultiScanBoilerPlate/android/app/src/main/java/com/ahireactnativemultiscanboilerplate/MultiScanModule.kt [hunk 1]
```diff
@@ -259,74 +382,30 @@ class MultiScanModule(private val context: ReactApplicationContext) :
...
-            writer.close()
-            true
-        } catch (e: Exception) {
-            print("AHI ERROR: KOTLIN: Exception when attempting to write file: $e")
-            false
+            scanType: String,
+            options: Map<String, Any>,
+            completionBlock: (result: AHIResult<Array<Map<String, Any>>>) -> Unit,
+        ) {
+            // TODO
```
Comment (shoaibcheema) [inline ReactNative/AHIReactNativeMultiScanBoilerPlate/android/app/src/main/java/com/ahireactnativemultiscanboilerplate/MultiScanModule.kt:397]: TODO ??

**Review by shoaibcheema — APPROVED** [review]
Looks good to me.

File: ReactNative/AHIReactNativeMultiScanBoilerPlate/android/app/src/main/java/com/ahireactnativemultiscanboilerplate/MultiScanModule.kt (hunk 1 above)
Comment (ChueiZuoChen) [reply ReactNative/AHIReactNativeMultiScanBoilerPlate/android/app/src/main/java/com/ahireactnativemultiscanboilerplate/MultiScanModule.kt:397]: implement v22 persistence delegate

**Pull request merged**